app/src/main/java/org/wikipedia/views/SurveyDialog.kt

```diff
@@ -55,7 +62,15 @@ object SurveyDialog {
         if (source == Constants.InvokeSource.RECOMMENDED_CONTENT) {
             dialogBuilder.setPositiveButton(R.string.patroller_diff_feedback_dialog_submit) { _, _ ->
                 val feedbackInput = binding.feedbackInput.text.toString()
-                // TODO: send event
+
+                ExperimentalLinkPreviewInteraction(source = HistoryEntry.SOURCE_SEARCH,
```

Comment:
Do we need to send `action_source =  43 OR 44` for personalized or generalized sources instead of a default source `1`? Same for other places which should we set `43` or `44` if the user sees the recommended content?